solutions/standard/provider.tf

```diff
@@ -4,3 +4,8 @@ provider "ibm" {
   region           = var.region
   ibmcloud_timeout = 60
 }
+
+provider "ibm" {
+  ibmcloud_api_key = var.ibmcloud_api_key
+  region           = var.region
+}
```

Comment:
I do not understand the requirement for this. Two provider blocks using the same key. We should probably stay with one provider block until work is completed to support KMS is a different account.

Reply:
The PR to add external KMS is merged - @kierramarie please rebase and address